README.md

```diff
@@ -16,7 +16,7 @@ so you can focus developing exciting features for your business case.
 - ensure docker & docker-compose is installed
 - Download the `installer.sh` from the latest release at https://github.com/spryker-sdk/sdk/releases
 - run `installer.sh </path/to/install/sdk/in>`
-- run `source ~/.zshrc` or `source ~/.bashrc` or re-open terminal
+- follow the prompts of the installer. Manual actions are possible in case of unsuccessful identification of the environment
```

Comment:
```
- follow the installer's instructions.
- alias `apryker-sdk` should be set and `SPRYKER_SDK_PATH` env variable should be exported. If not check our troubleshooting doc.
```
And then create `docs/troubleshooting.md` file and provide a solution there.

Example
```
# Troubleshooting

## `spryker-sdk` is not exist
`spryker-sdk` is an alias that uses {sdk_folder}/bin/spryker-sdk.sh script.
If `spryker-sdk` alias doesn't exist, please add it manually to your shell rc file. Depending on your shell it can be `.bashrc`, `.zshrc`, `.profile`, etc.".
`echo alias spryker-sdk={sdk_folder}/bin/spryker-sdk.sh` 
Run `source {rc file}` to load spryker-sdk alias for the current session.
```